nncf/common/factory.py

```diff
@@ -68,5 +75,25 @@ def create(model: TModel) -> Engine:
         if model_backend == BackendType.ONNX:
             from nncf.onnx.engine import ONNXEngine
             return ONNXEngine(model)
+        if model_backend == BackendType.OPENVINO:
+            from nncf.experimental.openvino_native.engine import OVNativeEngine
+            return OVNativeEngine(model)
         raise RuntimeError('Cannot create backend-specific engine'
                            'because {} is not supported!'.format(model_backend))
+
+
+# TODO(l-bat): Remove after fixing ticket: 100919
+def copy_model(model: TModel) -> TModel:
```

Comment:
It looks like the location of this method is out of place here. Could you suggest a new location?

Reply:
moved to `nncf/common/utils/backend`